src/framework/components/camera/component.js

```diff
@@ -176,12 +178,9 @@ class CameraComponent extends Component {
     set priority(newValue) {
         this._priority = newValue;
 
-        var layers = this.layers;
-        for (var i = 0; i < layers.length; i++) {
-            var layer = this.system.app.scene.layers.getLayerById(layers[i]);
-            if (!layer) continue;
-            layer._sortCameras();
-        }
+        // layer composition needs to update order
+        let layerComp = this.system.app.scene.layers;

```

Comment:
tiny point, but no need for layerComp variable.

Reply:
I agree, but I added it to make the code more readable as it's not clear from naming that scene.layers is LayerComposition.